### DISCUSSION TITLE
v3.0.83

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#9261] [General]
## What's Changed
* Update Korean (ko) translations by @ZerglingGo in https://github.com/filamentphp/filament/pull/9236
* Fix/missing dayjs translations by @pDonatas in https://github.com/filamentphp/filament/pull/9245
* Support `Htmlable` in global search result title by @AyoobMH in https://github.com/filamentphp/filament/pull/9240
* Re-order state formatting by @austincarpenter in https://github.com/filamentphp/filament/pull/9253
* Refactor avatars by @zepfietje in https://github.com/filamentphp/filament/pull/9258
* feature: Filter modal by @danharrin in https://github.com/filamentphp/filament/pull/9259

## New Contributors
* @ZerglingGo made their first contribution in https://github.com/filamentphp/filament/pull/9236
* @pDonatas made their first contribution in https://github.com/filamentphp/filament/pull/9245
* @AyoobMH made their first contribution in https://github.com/filamentphp/filament/pull/9240

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.0.82...v3.0.83

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.0.83'>v3.0.83</a>.</em>

**Comment by Umar-Farooq-Shafi** [comment]
The pagination is not showing anymore?

<img width="1062" alt="Screen Shot 2023-10-24 at 1 29 44 PM" src="https://github.com/filamentphp/filament/assets/62417503/12585a82-bc30-4ea4-9243-ab5a583a422d">

**Reply by danharrin** [reply]
Run `npm run build`

**Reply by Umar-Farooq-Shafi** [reply]
yes, I do run multiple times. it's not evening showing on the development

**Reply by Umar-Farooq-Shafi** [reply]
using `npm run dev`

**Reply by Umar-Farooq-Shafi** [reply]
I'm on `v3.0.83`


<img width="440" alt="Screen Shot 2023-10-24 at 3 25 06 PM" src="https://github.com/filamentphp/filament/assets/62417503/f5c0396b-d04a-4f7b-bc89-a521a00a8d90">

**Reply by Umar-Farooq-Shafi** [reply]
Also, the loading keeps showing on the widget filters

<img width="363" alt="Screen Shot 2023-10-24 at 3 29 37 PM" src="https://github.com/filamentphp/filament/assets/62417503/70dd3124-f79a-4e96-ab14-6514af7d74fe">

**Reply by danharrin** [reply]
Make sure Livewire is up to date. Run `filament:upgrade` too. No-one else has reported this, so I think it's probably an issue on your end.

**Reply by Umar-Farooq-Shafi** [reply]
Do I've to run this on every build?

**Reply by danharrin** [reply]
`filament:upgrade` should be added to your composer.json by `filament:install`, to ensure that it runs after you update Composer automatically.

**Reply by Umar-Farooq-Shafi** [reply]
@danharrin it's already in the `composer.json`

<img width="643" alt="Screen Shot 2023-10-24 at 6 00 14 PM" src="https://github.com/filamentphp/filament/assets/62417503/abd7cf88-b197-4747-8023-00fd4b9d20de">

**Reply by danharrin** [reply]
Okay, it should be run then

**Reply by sinnbeck** [reply]
Maybe try `rm -r ./vendor` and the do a `composer install` again

Also cannot recreate the problem in my own project

**Reply by Umar-Farooq-Shafi** [reply]
@sinnbeck didn't work that. I tried on a different system but it still same issue.

**Reply by danharrin** [reply]
Create a reproduction repository in a new app and open an issue with it

**Reply by Umar-Farooq-Shafi** [reply]
the issue was resource published files. after deleting the issue resolve. thanks
